src/partials/footer.html

```diff
@@ -1,5 +1,3 @@
-<footer>
-  <div class="container">
-    
-  </div>
+<footer id="footer">
+  <div class="container"></div>
```

Comment:
I think you should do any changes there. Since it can affect someone work on this section. And you will have conflicts